### PR TITLE
Add Addons namespace to Custom Ops

### PR DESCRIPTION
**Pull request by tomerk**
Fixes issue #379 
https://github.com/tensorflow/addons/issues/379

This requires the nightly or the upcoming TF2 RC2, it does not work with the TF2 RC1

### CODE REVIEW

**Comment by seanpmorgan** [comment]
Thanks very much Tomer this is a big help. Mind quickly formatting with:
`bash tools/run_docker.sh -c 'make code-format'`

**Comment by tomerk** [comment]
Done!

**Comment by tomerk** [comment]
Done, but it looks like the way I rebased onto master messed up the CLA bot.

**Comment by seanpmorgan** [comment]
> Done, but it looks like the way I rebased onto master messed up the CLA bot.

No worries we can override CLA when time to merge.

File: tensorflow_addons/custom_ops/activations/cc/kernels/gelu_op.h [hunk 1]
```diff
@@ -137,8 +138,7 @@ void GeluGradOp<Device, T>::OperateNoTemplate(OpKernelContext* context,
           approximate, output->flat<T>());
 }
 
-}  // namespace tensorflow
+}  // end namespace addons
+}  // namespace tensorflow#undef EIGEN_USE_THREADS
 
-#undef EIGEN_USE_THREADS
```
Comment (fsx950223) [inline tensorflow_addons/custom_ops/activations/cc/kernels/gelu_op.h]: why remove #undef EIGEN_USE_THREADS

**Comment by facaiy** [comment]
Nice job, Tomer!

File: tensorflow_addons/custom_ops/activations/cc/kernels/gelu_op.h (hunk 1 above)
Comment (seanpmorgan) [reply tensorflow_addons/custom_ops/activations/cc/kernels/gelu_op.h]: @tomerk When you get a moment could you confirm if this was or was not a mistake? After this issue we should be ready to merge.

Comment (tomerk) [reply tensorflow_addons/custom_ops/activations/cc/kernels/gelu_op.h]: Whoops, this should be fixed now!

**Comment by facaiy** [comment]
@ewilderj Hi, Edd, could you help override cla status? Thanks 😄

**Comment by facaiy** [comment]
@WindQAQ @fsx950223 Hi, could you take another look? I'd like to merge it if you both agree :-)

**Comment by facaiy** [comment]
Thanks, all!

**Pull request merged**